example_envs/bookinfo/gateway.rs

```diff
@@ -20,22 +25,54 @@ impl fmt::Display for Gateway {
 
 impl SimElement for Gateway {
     fn tick(&mut self, tick: u64) -> Vec<Rpc> {
-        let mut outbound_rpcs = self.core_node.tick(tick);
-        for outbound_rpc in &mut outbound_rpcs {
-            outbound_rpc
-                .headers
-                .insert("direction".to_string(), "request".to_string());
-            outbound_rpc
-                .headers
-                .insert("src".to_string(), self.core_node.id.to_string());
-            outbound_rpc
-                .headers
-                .insert("dest".to_string(), "productpage-v1".to_string());
+        log::error!("{:?}", self.core_node.generation_rate);
```

Comment:
Should we be logging this as an error?  It looks like it's just info.

Reply:
Yeah that should not be there at all.